### PR TITLE
Can mark games as played

### LINKED ISSUE #74: As a forgetful gamer, I want to be able mark games as played, so that I know if I've played it before
- [x] Save games played in database
- [x] Checkbox to fill, "Have you played this game?"

### PR DESCRIPTION
**Pull request by x183**
Added functionality to mark a game as played for a user, fixes #74

### CODE REVIEW

**Review by Oscariremma — CHANGES_REQUESTED** [review]

File: backend/src/services/gameService.ts [hunk 1]
```diff
@@ -144,3 +145,19 @@ export const removeGame = async (gameID: string) => {
 		}
 	});
 };
+
+export const markGameAsPlayed = async (gameID: string, cid: string) => {
+	console.log(`${cid} played ${gameID}`);
```
Comment (x183) [inline backend/src/services/gameService.ts:150]:
```suggestion
```

**Review by Oscariremma — APPROVED** [review]
Looks good!

File: backend/prisma/schema.prisma [hunk 2]
```diff
@@ -20,15 +20,25 @@ model Game {
...
+    borrow          Borrow[]
+    request         BorrowRequest[]
+    GameOwner       GameOwner?      @relation(fields: [gameOwnerId], references: [id])
+    gameOwnerId     String?
 
     @@map("games")
 }
 
+model PlayStatus {
+    id     String @id @default(cuid())
```
Comment (GAsplund) [inline backend/prisma/schema.prisma:33]: Should we be using an ID on something that can already be identified with just a `User` and `Game`? I don't see those values having reason to change down the line, so unless we're planning to remove an entry entirely to mark a player as no longer having played it, I dont see its use.

**Review by GAsplund — CHANGES_REQUESTED** [review]

**Review by GAsplund — APPROVED** [review]
LGTM!

File: backend/prisma/schema.prisma (hunk 2 above)
Comment (x183) [reply backend/prisma/schema.prisma]: Yes

**Pull request merged**